corehq/messaging/scheduling/models/content.py

```diff
@@ -1,16 +1,25 @@
 import jsonfield
+from corehq.apps.sms.api import send_sms_with_backend_name
 from corehq.messaging.scheduling.models.abstract import Content
+from corehq.apps.reminders.util import get_one_way_number_for_recipient
 from django.db import models
 
 
 class SMSContent(Content):
     message = jsonfield.JSONField(default=dict)
 
     def send(self, recipient):
-        print '*******************************'
-        print 'To:', recipient
-        print 'Message: ', self.message
-        print '*******************************'
+        phone_number = get_one_way_number_for_recipient(recipient)
+        if not phone_number or len(phone_number) <= 3:
```

Comment:
i saw a lot of cases with just the country code as their phone number, and they end up taking up a lot of processing power which we can avoid. since we know numbers less than 3 digits would never be real numbers, we can just ignore them.